router.go

```diff
@@ -238,9 +251,19 @@ func StringErrorResponse(err string) APIResponse {
 	return APIResponse{Error: err, Status: "ERROR", Data: nil}
 }
 
-// ErrorResponse constructs error response from the API
+// ErrorResponse constructs error response from the API with error level log
 func ErrorResponse(err error) APIResponse {
-	return APIResponse{Error: err.Error(), Status: "ERROR", Data: nil}
+	return APIResponse{Error: err.Error(), Status: "ERROR", Data: nil, Type: LogTypeError}
+}
+
+// ErrorInfoResponse constructs error response from the API with info level log
+func ErrorInfoResponse(err error) APIResponse {
```

Comment:
Can't we have these method?

`func ErrorInfoResponse(err error) APIResponse {
	return APIResponse{INFO: err.Error(), Status: "INFO", Data: nil, Type: LogTypeInfo}
}`

Reply:
@bidur-phil  Using `Status: "INFO"` will make this backward incompatible as in multiple instances in frontend we check `Status == "ERROR"`. 
This change would mean that we would need to change the FE too and test every integration to FE.

Another point to note is that even if it's just an `INFO` for grafana logs it's actually an `ERROR` for the FE.

Reply:
If we intend on changing the status and plan on changing the FE code I  would suggest that instead of sending `INFO` , `ERROR` or `WARN` in status we follow REST API principle and use status code instead.

Reply:
sounds good @Swechhya that could be separate project later.